src/controllers/enterpriseController.ts

```
@@ -1,88 +1,94 @@
+ * - reconciliation: Status and warnings for data consistency
  */
 export async function getTreasury(
-  _req: Request,
+  req: Request,
   res: Response,
   next: NextFunction,
 ): Promise<void> {
   try {
+    const organizationId = (req as any).apiKey?.organizationId || undefined;
+    const toleranceStr = req.query.tolerance as string | undefined;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)